modules/deck.gl/src/utils/series-color-utils.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2022, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {IndexType, Series, Uint32, Uint32Series} from '@rapidsai/cudf';
```

Comment:
This is failing in CI because `@rapidsai/cudf` needs to be added to the dependencies in `package.json`

Reply:
resolved, thanks!